<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UdpLogViewerClass</class>
 <widget class="QMainWindow" name="UdpLogViewerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Udp Log Server by gunoodaddy</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="topWindowCheck">
          <property name="text">
           <string>Top Window</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="tabPinCheck">
          <property name="text">
           <string>Tab Pin</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="scrollPinCheck">
          <property name="text">
           <string>Scroll Pin (F2)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="tabAllClearBtn">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Tab All Crear</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="logClearBtn">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Log Clear</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cssReloadBtn">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>CSS Reload</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="transparentSlider">
          <property name="minimum">
           <number>20</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="wordWrapCheck">
          <property name="text">
           <string>Word Wrap</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ignoreNewLineCheck">
          <property name="text">
           <string>Ignore New Line</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="searchText"/>
        </item>
        <item>
         <widget class="QPushButton" name="clearButton">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="prevBtn">
          <property name="text">
           <string>Prev (F3)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="nextBtn">
          <property name="text">
           <string>Next (F4)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="caseSensitiveCheck">
          <property name="text">
           <string>Case Sensitive</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="udplogviewer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>logClearBtn</sender>
   <signal>clicked()</signal>
   <receiver>UdpLogViewerClass</receiver>
   <slot>onClickedLogClear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabAllClearBtn</sender>
   <signal>clicked()</signal>
   <receiver>UdpLogViewerClass</receiver>
   <slot>onClickedTabAllClear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cssReloadBtn</sender>
   <signal>clicked()</signal>
   <receiver>UdpLogViewerClass</receiver>
   <slot>onClickedCSSReload()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>transparentSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>UdpLogViewerClass</receiver>
   <slot>onValueChangedWindowTransparent(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>461</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scrollPinCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>UdpLogViewerClass</receiver>
   <slot>onClickedScrollPin(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>topWindowCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>UdpLogViewerClass</receiver>
   <slot>onClickedTopWindow(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchText</sender>
   <signal>textEdited(QString)</signal>
   <receiver>UdpLogViewerClass</receiver>
   <slot>onChangedSearchText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchText</sender>
   <signal>textChanged(QString)</signal>
   <receiver>UdpLogViewerClass</receiver>
   <slot>onChangedSearchText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>caseSensitiveCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>UdpLogViewerClass</receiver>
   <slot>onClickedCaseSensitive(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>600</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>prevBtn</sender>
   <signal>clicked()</signal>
   <receiver>UdpLogViewerClass</receiver>
   <slot>onClickedSearchPrev()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>422</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchText</sender>
   <signal>returnPressed()</signal>
   <receiver>UdpLogViewerClass</receiver>
   <slot>onClickedSearchNext()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextBtn</sender>
   <signal>clicked()</signal>
   <receiver>UdpLogViewerClass</receiver>
   <slot>onClickedSearchNext()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>503</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wordWrapCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>UdpLogViewerClass</receiver>
   <slot>onClickedWordWrap(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ignoreNewLineCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>UdpLogViewerClass</receiver>
   <slot>onClickedIgnoreNewLine(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabWidget</sender>
   <signal>currentChanged(int)</signal>
   <receiver>UdpLogViewerClass</receiver>
   <slot>onTabCurrentChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>UdpLogViewerClass</receiver>
   <slot>onClickedClearSearchText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onClickedTabAllClear()</slot>
  <slot>onClickedLogClear()</slot>
  <slot>onClickedCSSReload()</slot>
  <slot>onValueChangedWindowTransparent(int)</slot>
  <slot>onClickedScrollPin(bool)</slot>
  <slot>onClickedTopWindow(bool)</slot>
  <slot>onChangedSearchText(QString)</slot>
  <slot>onClickedSearchPrev()</slot>
  <slot>onClickedSearchNext()</slot>
  <slot>onClickedCaseSensitive(bool)</slot>
  <slot>onClickedWordWrap(bool)</slot>
  <slot>onClickedIgnoreNewLine(bool)</slot>
  <slot>onTabCurrentChanged(int)</slot>
  <slot>onClickedClearSearchText()</slot>
 </slots>
</ui>
